tests/concepts/t3330.nim

```diff
@@ -5,29 +5,29 @@ t3330.nim(70, 4) Error: type mismatch: got <Bar[system.int]>
 but expected one of:
```

Comment:
how about coloring paths consistently in 1 color? eg:
`nim r --colors tests/misc/t9039.nim`
shows 
![image](https://user-images.githubusercontent.com/2194784/125869484-11f4b798-3289-4940-bad4-68d08f69d0e5.png)

=> can the `../../lib/system/basic_types.nim(2, 3)` be colored like `t9039.nim(30, 22)` ?

before doing anything, is that hard to do with what you have?

Reply:
Could be done in the declared locs using the `colors` as a control to toggle it.